docs/design/3_1_5_communications.tex

```diff
@@ -1 +1,35 @@
-TODO: Why WiFi? Do we need an external router? Throughput/bandwidth considerations? Why not combine it with drone control?
\ No newline at end of file
+The Air-Ground communication between the Raspberry Pi and the Base Station transmits video, ML results and control commands (to the computing platform). It is a bi-directional, high bandwidth communication channel that requires stable connection range for up to 30 meters.
```

Comment:
Make sure to link the relevant requirement numbers when you list a requirement. The requirement numbers might shift a bit over time, but we can update them at the end.